<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ABMWEClass</class>
 <widget class="QMainWindow" name="ABMWEClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>567</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ABMWE</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeView" name="treeView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="MainWindowView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave_Project"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionPan"/>
    <addaction name="actionReturn_to_Prev"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuABM">
    <property name="title">
     <string>ABM</string>
    </property>
    <addaction name="actionConfigure_Model"/>
    <addaction name="actionRun_Model"/>
    <addaction name="actionPause"/>
    <addaction name="actionReset_Model"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionImport_Data"/>
    <addaction name="actionExport"/>
    <addaction name="actionExport_Map"/>
   </widget>
   <widget class="QMenu" name="menuTriggers">
    <property name="title">
     <string>Triggers</string>
    </property>
    <addaction name="actionTrigger_Buffer"/>
    <addaction name="actionTrigger_Points"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout_ABMWE"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuData"/>
   <addaction name="menuABM"/>
   <addaction name="menuTriggers"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Project"/>
   <addaction name="actionOpen_Project"/>
   <addaction name="actionSave_Project"/>
   <addaction name="separator"/>
   <addaction name="actionImport_Data"/>
   <addaction name="actionExport_Map"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionPan"/>
   <addaction name="actionReturn_to_Prev"/>
   <addaction name="separator"/>
   <addaction name="actionConfigure_Model"/>
   <addaction name="actionRun_Model"/>
   <addaction name="actionPause"/>
   <addaction name="actionReset_Model"/>
   <addaction name="separator"/>
   <addaction name="actionTrigger_Buffer"/>
   <addaction name="actionTrigger_Points"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset>
     <normalon>:/ABMWE/Resources/newfile.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="icon">
    <iconset>
     <normalon>:/ABMWE/Resources/openfile.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="icon">
    <iconset>
     <normalon>:/ABMWE/Resources/savefile.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset>
     <normalon>:/ABMWE/zoomin.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset>
     <normalon>:/ABMWE/zoomout.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionReturn_to_Prev">
   <property name="icon">
    <iconset>
     <normalon>:/ABMWE/return.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Return to Prev</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Backspace</string>
   </property>
  </action>
  <action name="actionConfigure_Model">
   <property name="icon">
    <iconset>
     <normalon>:/ABMWE/configureModel.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Configure Model</string>
   </property>
  </action>
  <action name="actionRun_Model">
   <property name="icon">
    <iconset>
     <normalon>:/ABMWE/run.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Run Model</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset>
     <normalon>:/ABMWE/pause.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Pause Simulation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionImport_Data">
   <property name="icon">
    <iconset>
     <normalon>:/ABMWE/importdata.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Import Data</string>
   </property>
  </action>
  <action name="actionReset_Model">
   <property name="icon">
    <iconset>
     <normalon>:/ABMWE/reset.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Reset Model</string>
   </property>
  </action>
  <action name="actionTrigger_Buffer">
   <property name="icon">
    <iconset>
     <normalon>:/ABMWE/buffer.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Trigger Buffer</string>
   </property>
  </action>
  <action name="actionTrigger_Points">
   <property name="icon">
    <iconset>
     <normalon>:/ABMWE/point1.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Trigger Points</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normalon>:/ABMWE/help.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout_ABMWE">
   <property name="text">
    <string>About ABMWE</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionExport_Map">
   <property name="icon">
    <iconset>
     <normalon>:/ABMWE/exportmap.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Export Map</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="icon">
    <iconset>
     <normalon>:/ABMWE/draghand.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Pan</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainWindowView</class>
   <extends>QGraphicsView</extends>
   <header>mainwindowview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="abmwe.qrc"/>
 </resources>
 <connections/>
</ui>
